### PR TITLE
Added authorization tests

### PR DESCRIPTION
**Pull request by denysord88**
@agozhiy could you please review changes?

### CODE REVIEW

File: src/test/java/pages/LoginNavigatePage.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+package pages;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class LoginNavigatePage extends BasePage {
+  @FindBy(partialLinkText = "Log In")
+  private WebElement loginButton;
+
+  public void openMainLoginPage() {
```
Comment (agozhiy) [inline src/test/java/pages/LoginNavigatePage.java]: Page methods that open a new page should return the corresponding page object.

File: src/test/java/pages/LoginNavigatePage.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package pages;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class LoginNavigatePage extends BasePage {
```
Comment (agozhiy) [inline src/test/java/pages/LoginNavigatePage.java]: This page is not needed at all. We can use NavigationPage instead

File: src/test/java/pages/LoginPage.java [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  private WebElement userPassword;
+  @FindBy(xpath = "/html/body/div[2]/div[2]/form/fieldset/div/p[3]/button")
+  private WebElement submitLogin;
+
+  public NavigatePage authenticate() {
+    userName.sendKeys(TestProperties.drillUserName);
+    userPassword.sendKeys(TestProperties.drillUserPassword);
+    submitLogin.click();
+    return getPage(NavigatePage.class);
+  }
```
Comment (agozhiy) [inline src/test/java/pages/LoginPage.java]: Please extract this method to Steps. Page objects shouldn't contain any logic.

File: src/test/java/pages/LogoutNavigatePage.java [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package pages;
+
+import initial.TestProperties;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class LogoutNavigatePage extends BasePage {
```
Comment (agozhiy) [inline src/test/java/pages/LogoutNavigatePage.java]: The page is not needed. Please use NavigatioPage.

File: src/test/java/pages/MainLoginPage.java [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package pages;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class MainLoginPage extends BasePage {
```
Comment (agozhiy) [inline src/test/java/pages/MainLoginPage.java]: Please rename pages MainLoginPage and LoginPage.
They have different purpose. MainLoginPage is for choosing auth method (Form, SPNEGO etc), while LoginPage is used for entering credentials for Form authentication.

File: src/test/java/testng/unsecure/BaseUnsecureTest.java [hunk 6]
```diff
@@ -19,15 +19,17 @@
...
 import org.testng.annotations.AfterSuite;
 import org.testng.annotations.BeforeSuite;
+import steps.AuthSteps;
 import testng.BaseTest;
 
 public class BaseUnsecureTest extends BaseTest {
   @BeforeSuite
   public void beforeSuite() {
-
+    AuthSteps.login();
```
Comment (agozhiy) [inline src/test/java/testng/unsecure/BaseUnsecureTest.java]: Add an option check for secured Drill.

**Review by agozhiy — CHANGES_REQUESTED** [review]

**Comment by denysord88** [comment]
Updated
@agozhiy please review

**Review by agozhiy — APPROVED** [review]

**Comment by agozhiy** [comment]
+1 LGTM

**Pull request merged**